src/view/com/util/post-embeds/VideoEmbedInner.tsx

```diff
@@ -67,16 +73,30 @@ export function VideoEmbedInner({}: {
     }
   })
 
+  let aspectRatio = 16 / 9
+
+  if (embed.aspectRatio) {
+    const {width, height} = embed.aspectRatio
+    aspectRatio = width / height
+    aspectRatio = clamp(aspectRatio, 1 / 1, 3 / 1)
+  }
+
   return (
     <Animated.View
-      style={[a.flex_1, a.relative]}
+      style={[a.flex_1, a.relative, {aspectRatio}]}
       ref={aref}
       collapsable={false}>
       <VideoView
         ref={ref}
         player={player}
         style={a.flex_1}
+        contentFit="contain"
         nativeControls={true}
+        accessibilityIgnoresInvertColors
+        accessibilityLabel={
+          embed.alt ? _(msg`Video: ${embed.alt}`) : _(msg`Video`)
```

Comment:
hmm, yea i guess we need to stick `video:` in front of this huh... might be weird to hear "Video: Video of a cat eating a tuna" but might just be the way it is. We can tweak later if we want after testing VO/TB with the final thing.

Reply:
Yeah really just guessing here, might be good to ask for feedback after release